skeyan/stock-portfolio
# Setup backend

**Changes:**
- Setup backend with Express and MongoDB
- Setup Express routing for user registration
- Connected Registration submit button on the frontend to the backend and database in order to register a new user
- Closes #5 and closes #6 

**To Test:**
- Run `npm start`. Navigate to the Registration page via the navbar. Upon pressing "Register," check the console to see the response body printed. The error or success message will be found in `data.message`.
